### DISCUSSION TITLE
生医药引用格式要求更新

### DISCUSSION THREAD

**Discussion by Xiaofeiff** [tuna/thuthesis#921] [Q&A]
[2024春季论文格式说明.pdf](https://github.com/tuna/thuthesis/files/14231172/2024.pdf)
根据最新的生医药格式要求，引用需要按照旧的cell格式，查了一下大家之前的讨论
\bibliographystyle{cell}
添加了这行后在overleaf里可以把文中的引用改为要求的格式，但是文末的list开头还是有编号，按院系要求应该没有编号
![image](https://github.com/tuna/thuthesis/assets/107990941/29a10843-cb0d-4ea9-bd3f-9ebbe24e0305)
请问大家有没有方便的去掉的方法，谢谢

**Comment by zepinglee** [comment]
目前模板还没有适配 `cell.bst`，我稍后处理一下。

这个 `cell.bst` 是跟 `cite` 宏包配套使用的，后者其实与模板默认加载的 `natbib` 不兼容。我也不知道他们以前是怎么编译通过的。

@Xiaofeiff 另外参考文献表的格式是按照校研究生院《学位论文写作指南》的悬挂缩进还是按照 Cell 的非悬挂？生院有没有正式的格式要求？

<img width="750" alt="Screenshot 2024-02-14 at 13 43 28" src="https://github.com/tuna/thuthesis/assets/12290822/63a0c886-70a8-4fcc-8f5e-1a1e6728184d">

10-1参考文献-著者出版年制（示例）.doc

<img width="691" alt="Screenshot 2024-02-14 at 13 44 55" src="https://github.com/tuna/thuthesis/assets/12290822/b9a34a6c-e673-4cb2-8119-a2ed0f9cdc09">

<img width="729" alt="Screenshot 2024-02-14 at 13 46 29" src="https://github.com/tuna/thuthesis/assets/12290822/13ef7ec0-e7d4-42bf-aa23-03e7632171aa">

**Reply by Xiaofeiff** [reply]
谢谢学长，悬挂缩进和非悬挂缩进都能通过格式审查。因为我们系还要求图片页基本必须有正文段落，不能几幅图在同一页没有段落，我今年怕来不及调整，先跳车到word模板了，非常感谢！

**Reply by zepinglee** [reply]
我试了一下，`natbib` 和 `cell.bst` 是兼容的，只需要设置 `\bibliographystyle{cell}` 默认使用 author-date 即可。

> 我们系还要求图片页基本必须有正文段落，不能几幅图在同一页没有段落，我今年怕来不及调整，先跳车到word模板了，非常感谢！

这个不难解决。`figure` 和 `table` 的位置参数使用 `htp` 或者 `!htp`，模板中也放松了 LaTeX 对浮动体的限制，应该不太容易产生浮动体页（float page，即图或表单独成页）。如果还有可以调整 `figure` 或 `table` 在代码中的位置。

https://github.com/tuna/thuthesis/blob/b1bd9ab297ed89e0323c04c4d477c349bc9187c5/thuthesis.dtx#L3525-L3532
